docs/source/references/schema.json

```diff
@@ -3333,19 +3333,9 @@
     ]
   },
   "RejectedTransaction": {
-    "Struct": [
-      {
-        "name": "payload",
-        "type": "TransactionPayload"
-      },
-      {
-        "name": "signatures",
-        "type": "SignaturesOf<TransactionPayload>"
-      },
-      {
-        "name": "rejection_reason",
-        "type": "TransactionRejectionReason"
-      }
+    "Tuple": [
+      "VersionedSignedTransaction",
+      "TransactionRejectionReason"
     ]
```

Comment:
As for JS SDK, working with structs is easier than with tuples in JS/TS. So I would prefer to have this instead:

```rust
struct RejectedTransaction {
  transaction: VersionedSignedTransaction,
  reason: TransactionRejectionReason
}
```

Reply:
It's also easier in Rust. You can destructure directly and with no ambiguity.

Reply:
> As for JS SDK, working with structs is easier than with tuples in JS/TS.

schema can be defined independently of Rust types



> It's also easier in Rust. You can destructure directly and with no ambiguity.

I applied your suggestions. Though for posterity check the comment [here](https://github.com/hyperledger/iroha/pull/3485#discussion_r1198785396).